deepwell-core/src/models/page.rs

```diff
@@ -77,4 +79,17 @@ impl Page {
     pub fn exists(&self) -> bool {
         self.deleted_at.is_none()
     }
+   
+    #[cfg(feature = "ftml_compat")]
+    pub fn create_pageinfo<'a, TScoring: Scoring>(&self, votes: &Votes) -> ftml::PageInfoOwned {
+        ftml::PageInfoOwned {
+            title: String::from(self.title()),
+            alt_title: match self.alt_title() { None => None, Some(s) => Some(String::from(s)) },
```

Comment:
Anytime you have `None => None` in a match you can replace it with `expr.map(...)`

In this case you can do `self.alt_title().cloned()`.